### ISSUE TITLE
[Windows][Portable][Identifiable] Make Ricochet Portable

### ISSUE THREAD

**Issue by killhamburger** [blueprint-freespeech/ricochet-refresh#115]
C:/Users/NSA/AppData/Local/ricochet-refresh/tor/

Could you please NOT to use computer's AppData folder

and use ./ricochet_data instead?

```
ricochet-refresh.exe
tor.exe
ricochet_data (directory)
```

Problem 1. Your app is not portable (can't carry with USB to other computer because of this)
Peoblem 2. Other evil software can know ricochet-refresh's existence by looking above folder.

**Comment by elandorr** [comment]
I very much second this. Why has this been changed in the first place? The original Ricochet is perfectly portable. This is an absolute 200% must-have-no-questions-asked. A simple signature-verifiable archive to hand out to everyone. Put on an encrypted USB drive/container/whatever.

Thanks for updating the Tor in Ricochet, but please let it be portable again, thanks.

**Comment by pospeselr** [comment]
You can specify the profile location with the first arg from comand line/startup script (see #119)

**Comment by m-simonelli** [comment]
In the next release (3.0.11) we will be shipping an NSIS installer for Windows users.

Perhaps another way to go about this could be shipping a *non portable* installer, but if users want a portable install, we can have the download as it is now use the current path of the binary?

**Comment by evilmennot** [comment]
> NSIS installer for Windows users


Have you tried 7zip's exe?

**Comment by m-simonelli** [comment]
> > NSIS installer for Windows users
> 
> Have you tried 7zip's exe?

Yes, however for ease of use, we have made an NSIS installer (and +it makes uninstallers super easy :p )

**Comment by m-simonelli** [comment]
I'm planning on resolving this issue in the coming month, @killhamburger @elandorr @woowagon @evilmennot what are your thoughts on this solution?: https://github.com/blueprint-freespeech/ricochet-refresh/issues/115#issuecomment-939429982. IMO I think this is a nice solution, but I'd like some end-user feedback before I go and implement it :)

**Comment by m-simonelli** [comment]
@killhamburger @elandorr @woowagon @evilmennot some feedback before I implement this would be very helpful - I don't particularly want to implement something if you aren't happy with it.

**Comment by elandorr** [comment]
@m-simonelli That sounds great!  Sorry for the delayed response, the governments covid terror, the police brutality, the hyperinflation.. life's still not getting any easier. Massive email backlog.
